### ISSUE TITLE
[Proposal] Adopt solarized (light) as the default terminal emulator theme

### ISSUE THREAD

**Issue by wesbluemarine** [Barrett17/V-OS#24]
The default theme of the V-OS terminal emulator is not suitable for coding/programming.
Solarized is a well-studied theme to avoid eye strain and is appreciated by a lot of programmers.

What about adopting it as the default theme?

**Comment by Barrett17** [comment]
Makes sense to me. While I'm not entirely sure about light vs dark. I had the impression that the darker has less eye strain?

**Comment by wesbluemarine** [comment]
Solarized Dark has less eye strain for sure, but Solarized Light **I think** could have higher readability and may be could be more appropriate as it is more yellowish.

**Comment by Barrett17** [comment]
It would be awesome to have the theme change depending on the time. I believe it should be easy to apply the color changes in real time.

**Comment by wesbluemarine** [comment]
After weeks of use I can say that, personally, I prefer Gruvbox (light) over Solarized.
Having Redshift in V-OS would be also really great.

**Comment by Barrett17** [comment]
Yeah, definitely would be great to have something like that. I don't mind about the theme actually, but I'd be careful as some themes may not work properly with commands with colored output.
